bin/rosie-disco

```diff
@@ -20,22 +20,19 @@
 # NAME
 #     rosie disco
 #
-# SYNOPSIS
-#     rosie disco start [PORT] # start ad-hoc web service server (on PORT)
-#     rosie disco stop         # stop ad-hoc web service server
-#     rosie disco stop -y      # stop ad-hoc web service server w/o prompting
-#     rosie disco              # print status of ad-hoc web service server
-#
 # DESCRIPTION
 #     Start/stop ad-hoc Rosie suite discovery web service server.
 #
-#     For `rosie disco start`, if `PORT` is not specified, use port 8080.
-#
 # OPTIONS
-#     --non-interactive, --yes, -y
-#         (For stop only.) Switch off interactive prompting (=answer yes to
-#         everything)
-#     --service-root, -R
-#         (For start only.) Include web service name under root of URL.
+#     --start
```

Comment:
This is quite a big change to the way the CLI is used. Is this deliberate?

Reply:
Well noticed. It is deliberate & I have tried to explain the background in the third bullet point down under the 'Context on changes' section in my opening comment to the PR (sorry, there is a lot of content in that comment, so it is easy to miss). I'm happy to go into more detail verbally if you would like, though.